### ISSUE TITLE
Can you publish guides about how can use it

### ISSUE THREAD

**Issue by Ravid-Levy** [cmusphinx/pocketsphinx#274]
Can you publish guides about how can use it in java language or other programming language instead of python?

And can you publish what are the changes that you are made in this repo?

Thanks
Ravid

**Comment by dhdaines** [comment]
No, because the Java interface code was unmaintained, I don't actually know how to use it, there wasn't any existing documentation, and I don't have time to learn in order to maintain it.  If someone else wants to do so, they are welcome!

**Comment by dhdaines** [comment]
First, ask yourself if you *really* want to run a command-line program on Windows to do live speech recognition from the microphone.  Is this a useful thing for you to do?  Tell me your use case and I will let you know how to do it.
